apps/src/music/player/SoundCache.ts

```diff
@@ -6,12 +6,21 @@ import {baseAssetUrlRestricted} from '../constants';
 import {LoadFinishedCallback} from '../types';
 
 class SoundCache {
+  private static instance: SoundCache;
```

Comment:
Personal preference, I usually don't use underscores for private members in typescript since we already have the `private` modifier. But can change if you think it's clearer!

Reply:
Not a personal preference, but saw it used in `Lab2Registry.ts` so thought we could be consistent. I have it used in my current PR https://github.com/code-dot-org/code-dot-org/blob/e7eeb24045b16bd27f9bd8aa7fb59aad7866edd8/apps/src/aichat/chatEventLogger.ts.
I can remove in my current PR for consistency.

Reply:
Gotcha - yeah I don't have a strong opinion either way. I'll merge this is as and maybe we can revisit as part of a broader code style discussion?